java/1.2/modules/ROOT/pages/configure-java-module-acb-edit.adoc

```diff
@@ -0,0 +1,331 @@
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Constructor*, search and select the constructor to use. 
++
+NOTE: If this is blank, click the refresh icon to populate it.
+. Select the *Advanced* tab to configure the following attributes: 
++
+[%header%autowidth.spread]
+|===
+| UI Field | Description | Required
+| Target variable | Name of the variable to assign the class instance to. | No
+| Target value | Expression that evaluates the operation's output. The expression outcome is stored in the target variable. | No
+| Error Mappings a| Creates custom error types that you map to Mule error types. The mappings can help you further differentiate and group errors in your app. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:mule-runtime::mule-error-concept.adoc[]. | No
+|===
+. Select the *Input/Output* tab to filter and view input and output variables stored in DataWeave.
+
+After you add your class instance, <<configuring-invoke,configure the *Invoke* operation>>.
+
+[[configuring-invoke]]
+=== Configuring the Invoke Operation
+
+The *Invoke* operation invokes methods with the provided arguments using a specified class instance. The identifier of the method to invoke includes the class and method names.
+
+To configure the attributes for *Invoke* in the Anypoint Code Builder UI:
+
+. Click the *Invoke* component on the canvas.
+. In the *General* tab:
+.. Select the class *Instance* for invoking the method.
++
+This is the target variable assigned to the *New* operation.
+.. In *Class*, select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Method*, search and select the method to use. 
++
+NOTE: If this is blank, click the refresh icon to populate it.
+include::connectors::partial$acb-java-module-partial.adoc[tag="advanced-tab-attributes"]
+
+[[configuring-invoke-static]]
+=== Configuring the Invoke Static Operation
+
+Invoke static methods with the arguments you provide. 
+
+To configure the attributes for *Invoke Static* in the Anypoint Code Builder UI:
+
+. Click the *Invoke Static* component on the canvas.
+. In the *General* tab:
+.. In *Class*, select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Method*, search and select the method to use. 
++
+Only the static methods from the class are displayed.
++
+NOTE: If this is blank, click the refresh icon to populate it.
+include::connectors::partial$acb-java-module-partial.adoc[tag="advanced-tab-attributes"]
+
+=== Configuring the Validate Type Operation
+
+Use this operation to validate that the instance is an `instanceof` the expected class.
```

Comment:
```suggestion
Use this operation to validate that the instance is an `instanceof` the expected class:
```

Reply:
In general, we don't use elements this way.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-UIelements

Suggest 

If you need to keep it, suggest something like:
Use this operation to validate that the instance is an instance of (`instanceof`) the expected class.